modules/http-servlet/src/main/java/org/glassfish/grizzly/servlet/SessionCookieConfig.java

```diff
@@ -16,19 +17,27 @@
 
 package org.glassfish.grizzly.servlet;
 
+import java.util.Collections;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.glassfish.grizzly.http.server.Constants;
+
+
 /**
  * Class that may be used to configure various properties of cookies used for session tracking purposes.
  */
 public class SessionCookieConfig implements jakarta.servlet.SessionCookieConfig {
 
-    private String name;
-    private String domain;
-    private String path;
-    private String comment;
-    private boolean httpOnly = true;
-    private boolean secure;
+    private static final int DEFAULT_MAX_AGE = -1;
+    private static final boolean DEFAULT_HTTP_ONLY = false;
```

Comment:
@dmatej could you tell me the reason for changing this default value?

Reply:
Good question, but I don't know after 3 months, perhaps I should make a comment in the commit. It seems to me as a mistake now.